docs/configuration/settings.rst

```diff
@@ -5,113 +5,472 @@ Mautic Configuration Settings
+
+* **Do Not Track 404 error for anonymous Contacts** - Select **Yes** to not track page hits on any 404 error page tracked by the tracking code. This option helps prevent filling your logs with hits from bots.
+  
+.. note:: 
+
+  * The tracking code automatically detects the Preferred Timezone and Preferred Locale fields.
+  * Pages including 4-byte UTF-8 characters, such as emojis and some Chinese or other non-Latin characters, in the page title or URL aren't tracked on a Contact’s activity history in Mautic. All Latin characters used in English and other western languages are of 1-byte and are tracked.
+
 Facebook pixel
 ==============
 
+.. image:: images/facebook-pixel-settings.png
+  :width: 600
+  :alt: Screenshot showing Facebook Pixel Settings Configuration in Mautic
+
+* **Facebook Pixel ID** - Enter your Facebook Pixel ID and select the options you’d like to use the pixel for.
+
+* **Enabled on your tracking page** - Select Yes to have Mautic append the Facebook Pixel to the Mautic tracking code to track pages where the tracking code exists.
+
+* **Enabled on Mautic landing page** - Select Yes to have Mautic add the Facebook Pixel to Mautic landing pages.
+
+
 Google Analytics
 ================
 
+.. image:: images/google-analytics-settings.png
+  :width: 600
+  :alt: Screenshot showing Google Analytics Settings Configuration in Mautic
+
+* **Google Analytics ID** - Enter your Google Analytics ID and select the options you’d like to use the pixel for.
+
+* **Enabled on your tracking page** - Select Yes to have Mautic append the Google Analytics script to the Mautic tracking code to track pages where the tracking code exists.
+
+* **Enabled on Mautic landing page** - Select Yes to have Mautic add the Google Analytics script to Mautic landing pages.
+
+* **Enabled IP Anonymization** - For subscribers sensitive to GDPR or other data privacy laws and regulations, select Yes to anonymize the IP address of web visitors before sending it to Google Analytics.* 
+
 Report settings
 ***************
 
+.. image:: images/report-settings.png
+  :width: 600
+  :alt: Screenshot showing Report Settings Configuration in Mautic
+
+* **Always quote data in CSV export** - Select Yes to wrap each Mautic field in double quotation marks when exported to a CSV file. For example: ``"First Name",”Last Name”,””, "some text"``.
+
 Text message settings
 *********************
 
+.. image:: images/text-message-settings.png
+  :width: 600
+  :alt: Screenshot showing Text Message Settings Configuration in Mautic
+
+* **Select default transport to use** - If you have configured a delivery service for SMS messages, select the service here to send messages. You must configure a delivery service before selecting it here.
```

Comment:
Please make a screenshot and instructions with an SMS transport set up, so that it's explained what the settings do. It is quite similar to the email ones, I think. I have a trial Twilio account which I use for the purpose, should not be too difficult to do.

Reply:
> We are definitely getting there!
> 
> Here are a few suggestions, but I also noticed a lot of Vale warnings in the Gitpod console (check the Problems tab) - too many for me to fix in a review.
> 
> Please take a look and let me know if you get stuck with any of the passive to active speech ones!

Most of the vale errors occur on the wording page. Should the wording page be replaced with Landing Page even though it's not a Mautic feature? For example, should the Login Page be replaced with Login Landing Page?

Reply:
Okay.